src/index.js

```diff
@@ -3,6 +3,16 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+let pathSet = false;
+function setPath() {
+  if(!pathSet) {
```

Comment:
I'm not seeing as `pathSet` variable would be used anywhere. I assumed that it tend to be a flag to set PATH only once. I think it is not needed as `call` function is called only once in the process.